doc/references.bib

```diff
@@ -422,6 +422,21 @@ @inproceedings{DharmapraniEtAl2016
   pages = {825--828},
 }
 
+@article{DonoghueEtAl2020,
+  title = {Parameterizing neural power spectra into periodic and aperiodic components},
+  volume = {23},
+  issn = {1097-6256, 1546-1726},
+  url = {https://www.nature.com/articles/s41593-020-00744-x},
+  doi = {10.1038/s41593-020-00744-x},
+  pages = {1655--1665},
+  number = {12},
+  journaltitle = {Nature Neuroscience},
```

Comment:
```suggestion
  journal = {Nature Neuroscience},
```
this should fix the CircleCI build failure. Lots of other CIs are red too (incl. style)...  let us know when it's ready for another round of review.